### PR TITLE
add squash and rebase command

### PR DESCRIPTION
**Pull request by lewurm**

### CODE REVIEW

**Comment by lewurm** [comment]
@luhenry @akoeplinger please have a look.

File: monors.py [hunk 1]
```diff
@@ -141,6 +141,15 @@ def has_merge_command (self):
...
         return False
 
+    def has_merge_command (self):
+        return self.has_command("merge")
+
+    def has_squash_command (self):
+        return self.has_command("squash")
+
+    def has_rebase_command (self):
+        return self.has_command("rebase")
```
Comment (akoeplinger) [inline monors.py]: I'm wondering whether we should call the new commands `squash and merge` and `rebase and merge` to be more aligned with what they're called in the GitHub merge button?

Comment (lewurm) [reply monors.py]: I think that would be more confusing.

But I guess we can add both?

Comment (akoeplinger) [reply monors.py]: I find it more confusing that `@monojenkins rebase` would also merge the PR, not just doing a rebase ;)

**Review by luhenry — APPROVED** [review]

File: monors.py [hunk 2]
```diff
@@ -141,6 +141,15 @@ def has_merge_command (self):
...
         return False
 
+    def has_merge_command (self):
+        return self.has_command("merge")
+
+    def has_squash_command (self):
+        return self.has_command("squash") or self.has_command("squash and merge")
+
+    def has_rebase_command (self):
+        return self.has_command("rebase") or self.has_command("rebase and merge")
```
Comment (lewurm) [inline monors.py:151]: @akoeplinger does that look okay for you?

**Review by akoeplinger — APPROVED** [review]

**Comment by lewurm** [comment]
@monojenkins rebase

😛

**Pull request merged**